front/components/assistant_builder/ActionScreen.tsx

```diff
@@ -641,6 +575,72 @@ export default function ActionScreen({
             canSelectTable={dataSources.length !== 0}
           />
         </ActionModeSection>
+
+        <ActionModeSection show={builderState.actionMode === "DUST_APP_RUN"}>
+          {noDustApp ? (
+            <ContentMessage
+              title="You don't have any Dust Application available"
+              variant="warning"
+            >
+              <div className="flex flex-col gap-y-3">
+                {(() => {
+                  switch (owner.role) {
+                    case "admin":
+                    case "builder":
+                      return (
+                        <div>
+                          <strong>
+                            Visit the "Developer Tools" section in the Build
+                            panel to build your first Dust Application.
+                          </strong>
+                        </div>
+                      );
+                    case "user":
+                      return (
+                        <div>
+                          <strong>
+                            Only Admins and Builders can build Dust
```

Comment:
Similar to copy comment above
"Contact an Admin or Builder if you need a Dust Application."